packages/scene-composer/src/components/three-fiber/DataOverlayComponent/__tests__/__snapshots__/DataOverlayContainerSnap.spec.tsx.snap

```diff
@@ -47,15 +47,6 @@ exports[`DataOverlayContainer should render with panel visible correctly when th
   <div
     style="overflow: auto; border-radius: 2px; box-shadow: 0px 1px 4px -2px rgba(0, 28, 36, 0.5); padding-top: 12px; padding-bottom: 12px; width: 210px; min-height: 2em;"
   >
-    <div
```

Comment:
If we remove this, then user won't be able to close the overlay if they want to. 
Is this approved by UX?

Reply:
The selection logic closes it instead.  That button is only present when 'Show All Overlays' isn't active, which in the new logic those overlays automatically close when their tag is no longer selected. So you now close by selecting something else instead of using the button. I did show the video without buttons to UX and call out that specific change.

Reply:
did we decide to change this behavior without a config that customer can turn on/off? also can clean up the styles for the close button

Reply:
Discussed not needing 3rd option. Removing dead css.